awstesting/performance/logging.go

```diff
@@ -0,0 +1,116 @@
+// Package smoke contains shared step definitions that are used across integration tests
+package performance
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbattribute"
+)
+
+// benchmarkLogger handles all benchmark logging
+type benchmarkLogger struct {
+	outputer
+}
+
+// logger interface that handles any logging to an output
+type logger interface {
+	log(key string, data map[string]interface{}) error
+}
+
+// init intializes the logger and uses dependecy injection for the
+// outputer
+func (b *benchmarkLogger) init(output string) {
```

Comment:
I think a `newBenchmarkLogger` which returns a `*benchmarkLogger` instead of an `init()` with string would be a little easier to follow. If there are multiple types of benchmark loggers I suggests split them into different types that share a common interface.

It looks like they kind of already do with `outputer` and `logger`  Could an outputter be a logger?